src/extensions/layout-selector/editor.scss

```diff
@@ -4,36 +4,6 @@ $color-highlight: #00a4a6;
 	opacity: 1 !important;
 }
 
-.coblocks-layout-selector .coblocks-layout-selector__layout .block-editor-block-preview__container.editor-styles-wrapper {
```

Comment:
Why did you remove these styles?

Reply:
Probably just did it to strip every possible override during experimentation. This will continue to exist.